lib/recurly/risk/three-d-secure/strategy/adyen.js

```diff
@@ -133,7 +134,7 @@ export default class AdyenStrategy extends ThreeDSecureStrategy {
       redirect_url: adyenRedirectParams.url,
       ...adyenRedirectParams.data
     };
-    this.frame = recurly.Frame({ type: 'iframe', path: '/three_d_secure/start', payload, container })
+    this.frame = recurly.Frame({ type: TYPE.IFRAME, path: '/three_d_secure/start', payload, container })
```

Comment:
Moving the enum will obviate the need to import `TYPES` separately

```suggestion
    this.frame = recurly.Frame({ type: Frame.TYPES.IFRAME, path: '/three_d_secure/start', payload, container })
```

Reply:
This was my initial implementation, but I'm getting a reference error with this in my test: 

![image](https://user-images.githubusercontent.com/32269552/65788871-b4807a80-e121-11e9-8270-92e62be2e526.png)

![image](https://user-images.githubusercontent.com/32269552/65788904-c6fab400-e121-11e9-8fe8-dfc436a6402e.png)


I tried a few other ways to access TYPES, but none of them worked. Any ideas?

Reply:
Oh right, we'd just need to import Frame into the `adyen.js` module

```js
import Frame from '../../../frame';
```

or reference it from the recurly instance in this routine

```js
this.frame = recurly.Frame({ type: recurly.Frame.TYPES.IFRAME, path: '/three_d_secure/start', payload, container });
```